src/geo/transform.js

```diff
@@ -1639,6 +1663,20 @@ class Transform {
         this.worldToFogMatrix = this._camera.getWorldToCameraPosition(cameraWorldSize, cameraPixelsPerMeter, windowScaleFactor);
     }
 
+    _calcDistanceMatrices() {
+        this._distanceTileMatrixCache = {};
+        const center = this.point;
+
+        const m = new Float64Array(16);
+        const windowScaleFactor = 1 / this.height;
+        mat4.fromScaling(m, [windowScaleFactor, -windowScaleFactor, windowScaleFactor]);
+        mat4.rotateZ(m, m, this.angle);
+        mat4.translate(m, m, [-center.x, -center.y, 0]);
+
+        this.worldToCenterDistanceMatrix = m;
+        this.mercatorCenterDistanceMatrix = mat4.scale([], m, [this.worldSize, this.worldSize, 1]);
```

Comment:
As per above comment on references of that matrix, it seems like we can remove that statement.